src/backend/address.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import deepEqual from "lodash.isequal";
+
+export type Address = {|
+  +repositoryName: string,
+  +pluginName: string,
+  +id: string,
+|};
+
+export interface Addressable {
+  +address: Address;
+}
+
+export type SansAddress<T: Addressable> = $Exact<$Diff<T, {+address: Address}>>;
```

Comment:
I assume it's not possible to re-use the Addressable interface in this expression.

Reply:
Correct.